src/common/collection.c

```diff
@@ -1502,6 +1502,27 @@ static gchar *get_query_string(const dt_collection_properties_t property, const
       query = dt_util_dstrcat(query, "(id %s group_id)", (strcmp(escaped_text, _("group leaders")) == 0) ? "=" : "!=");
       break;
 
+    case DT_COLLECTION_PROP_MODULE: // dev module
+      {
+        dt_iop_set_module_name_table();
```

Comment:
We want to call this only once at startup I think. So better place it during initialization of the application in darktable.c.

Reply:
I would propose at the end of dt_init() routine at least after the call to   dt_iop_load_modules_so();

Reply:
It's what I've done. But this doesn't solve that the list is not ready when collect.c refreshes the screen.

Reply:
sorry, not collect.c but dt_collection_update()

Reply:
dt_collection_update() is called by dt_collection_new() and dt_selection_new()
around line 900 in darktable.c

Reply:
So let's try to move both calls around line 980 after dt_iop_load_modules_so(); It should work.

Reply:
seg fault ...
I've tried to call dt_collection_update(darktable.collection) after updating the list.
I get this:
![image](https://user-images.githubusercontent.com/23012047/74589195-f0897e80-4fe1-11ea-9ef5-8997d2ebaacd.png)
Like a kind of redraw was missing ...

Reply:
but   dt_control_queue_redraw_center(); doesn't help